java/code/src/com/redhat/rhn/common/db/datasource/xml/System_queries.xml

```diff
@@ -2299,6 +2299,14 @@ ORDER BY E.update_date, E.id
   </query>
 </mode>
 
+<callable-mode name="fail_action_for_system">
```

Comment:
True, write-mode is more suitable for this, but no longer needed. I have decided to do it another way. See cb43a5584a766504e5dbabb6abe056b32652d38b